### PR TITLE
Throw error if setting upload status as completed fails

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

File: npm/src/updateconsignmentstatus/index.ts [hunk 1]
```diff
@@ -18,23 +19,28 @@ export class UpdateConsignmentStatus {
...
+      if (!result.data || !result.data.markUploadAsCompleted || result.errors) {
+        const errorMessage: string = result.errors
+          ? result.errors.toString()
+          : "no data"
+        throw Error(errorMessage)
+      } else {
+        return result.data.markUploadAsCompleted
+      }
+    } catch (e) {
+      handleUploadError(
```
Comment (suzannehamilton) [inline npm/src/updateconsignmentstatus/index.ts:40]: This function is about calling the API, so making it call a method which changes the UI mixes up the levels of abstraction, which I think will make it harder to find this code. Where is the `markConsignmentStatusAsCompleted` method called, and does it deal with code at the page level? Would moving the `catch` and the call to `handleUploadError` fit better there?

Comment (suzannehamilton) [inline npm/src/updateconsignmentstatus/index.ts:40]: Can this try/catch be deleted now that upload/index.ts calls `handleUploadError`?

Comment (techncl) [reply npm/src/updateconsignmentstatus/index.ts:40]: 🤦  ah, I thought I deleted it. Thanks

**Review by suzannehamilton — APPROVED** [review]
🎉

**Pull request merged**